### PR TITLE
Completed css-canvas-medical challenge

### PR DESCRIPTION
**Pull request by CTMendoza**
canvas-medical

### CODE REVIEW

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+.col-one-half {
+  flex-basis: calc(100% / 2);
+}
+
+.col-one-third {
+  flex-basis: 360px;
```
Comment (BrettAlbright) [inline css-canvas-medical/styles.css]: this needs to be a percentage based value, should be calc(100%/3)

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
one CSS update needed, also, check the `clinicians prefer canvas` `ready to learn more` and bottom header to make sure the text alignment matches the example. 

Overall looking really good! Just make sure the spacing and placement matches the example.

File: css-canvas-medical/styles.css (hunk 1 above)
Comment (CTMendoza) [reply css-canvas-medical/styles.css]: canvas-medical

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**